onfido/resources/autofill.py

```diff
@@ -0,0 +1,6 @@
+from ..resource import Resource
+
+
+class Autofill(Resource):
+    def perform(self, request_body: dict):
```

Comment:
The rest of the methods in the library accepts the arguments directly, not as a dict - for example get applicant takes applicant_id and constructs request body inside. Autofill endpoint can only take `document_id` as a parameter so I would suggest reworking it to `def perform(self, document_id: string)`

Reply:
Cool, thank you for the suggestion @kulak-at. Have renamed the class as well to make it consistent with other libraries.